frontend/Themes/Themes.js

```diff
@@ -450,6 +474,7 @@ module.exports = {
   Eighties,
   FirefoxDark,
   FirefoxLight,
+  Firebug,
```

Comment:
That's what I was questioning, too.  I think, whichever decision we go with, the displayName and the key should reflect each other.  If we use `FirefoxFirebug`, I think the displayName should be either `Firefox Firebug` or `Firefox (Firebug)`.